README.md

```diff
@@ -275,3 +260,192 @@ datetime
 2018-06-04 00:00:00+00:00  0.07947329054545459  602.5116327272724
 
 ```
+
+### Exchange funds flow
+
+Fetch the difference between the tokens that were put in minus the tokens that were taken out from an exchange for a given slug in the selected time period.
+
+```python
+
+prices = san.get(
+    "exchange_funds_flow/santiment",
+    from_date="2018-04-16T10:02:19Z",
+    to_date="2018-05-23T10:02:19Z"
+)
+
+```
+
+Example result:
+
+```bash
+
+                              fundsFlow
+datetime
+2018-04-16 10:02:19+00:00      0.000000
+2018-04-16 10:17:18+00:00      0.000000
+2018-04-16 12:03:51+00:00      0.000000
+2018-04-16 13:50:24+00:00   -208.797310
+
+```
+
+### ERC20 Exchange Funds Flow
```

Comment:
I am wondering if we should name this a bit differently. It is a kind of an overview of the exchange funds flow + as we add more blockchains it won't cover only ERC20, but all blockchains. May be `Exchange Funds Flow Overview`?

Reply:
I agree but let's do this in a separate task because this will need changes in tech indicators, grafana, sanbase and sanpy.

Reply:
😢 seems like we need to do quite a bit of planning around naming things that will be included in the public APIs...